roles/ceph-common/defaults/main.yml

```diff
@@ -130,6 +130,16 @@ ceph_stable_rh_storage_iso_install: false # usually used when nodes don't have a
 ceph_stable_rh_storage_mount_path: /tmp/rh-storage-mount
 ceph_stable_rh_storage_repository_path: /tmp/rh-storage-repo # where to copy iso's content
 
+
+# UBUNTU CLOUD ARCHIVE
+# This allows the install of Ceph from the Ubuntu Cloud Archive.  The Ubuntu Cloud Archive
+# usually has newer Ceph releases than the normal distro repository.
+#
+ceph_stable_uca: false
+ceph_stable_repo_uca: "http://ubuntu-cloud.archive.canonical.com/ubuntu"
+ceph_stable_openstack_release_uca: liberty
```

Comment:
I think those 2 should not be enabled. Otherwise it'll always try to install from Ubuntu.